internal/datastore/postgres/revisions_test.go

```diff
@@ -83,6 +84,23 @@ func TestRevisionSerDe(t *testing.T) {
 	}
 }
 
+func TestTxIDTimestampAvailable(t *testing.T) {
+	testTimestamp := uint64(time.Now().Unix())
+	pgr := postgresRevision{snapshot: pgSnapshot{}, optionalTxID: newXid8(1), optionalTimestamp: testTimestamp}
+	receivedTimestamp, ok := pgr.OptionalTimestamp()
+	require.True(t, ok)
+	require.Equal(t, receivedTimestamp, testTimestamp)
+	txid, ok := pgr.OptionalTransactionID()
+	require.True(t, ok)
+	require.Equal(t, newXid8(1), txid)
+
+	pgr = postgresRevision{snapshot: pgSnapshot{}}
+	_, ok = pgr.OptionalTimestamp()
+	require.False(t, ok)
+	_, ok = pgr.OptionalTransactionID()
+	require.False(t, ok)
```

Comment:
Add an assertion that they are still equal here, so long as the snapshot is the same

Reply:
done